<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-V9DUL7Q0" failures="2" tests="2" name="Default test(failed)" time="0.077" errors="0" timestamp="2023-08-14T17:27:59 IST">
  <testcase classname="SeleniumInterviewQuestions.FailedTestCheck" name="testcase1" time="0.02">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at SeleniumInterviewQuestions.FailedTestCheck.testcase1(FailedTestCheck.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- testcase1 -->
  <testcase classname="SeleniumInterviewQuestions.FailedTestCheck" name="testcase2" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at SeleniumInterviewQuestions.FailedTestCheck.testcase2(FailedTestCheck.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- testcase2 -->
</testsuite> <!-- Default test(failed) -->
